source/_components/light.mqtt_json.markdown

```diff
@@ -52,6 +54,8 @@ Configuration variables:
 - **state_topic** (*Optional*): The MQTT topic subscribed to receive state updates.
 - **brightness** (*Optional*): Flag that defines if the light supports brightness. Default is false.
 - **rgb** (*Optional*): Flag that defines if the light supports RGB colors. Default is false.
+- **rgb_value_template** (*Optional*): Template to extract the color in the format 'R,G,B' from the state topic JSON. Defaults to `{{ value_json.color.r }},{{ value_json.color.g }},{{ value_json.color.b }}`.
```

Comment:
If you want to show templates then you have to wrap them in `{% raw %}...{% endraw %}` otherwise they will appear blank after rendering.